cluster/examples/openshift-bridge-marker.yml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: bridge-marker
+  namespace: kube-system
+  labels:
+    tier: node
+    app: bridge-marker
+spec:
+  selector:
+    matchLabels:
+      tier: node
+      app: bridge-marker
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: bridge-marker
+    spec:
+      serviceAccountName: bridge-marker
```

Comment:
I think just like the ovs make it will be nice if the daemonset can deploy both bridge-cni and the bridge-marker

Reply:
I agree, but it should not be done in this project (operator is a better place). With OVS we host both CNI and marker, this should stay for the marker only. We can document how to deploy both though.

Reply:
To be changed in cluster-network-addons-operator